src/grafana_dashboards/opensearch.json

```diff
@@ -2630,30 +2630,78 @@
         "type": "custom"
       },
       {
-        "datasource": {
-          "type": "datasource",
-          "uid": "${prometheusds}"
+        "allValue": ".*",
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
         },
+        "datasource": "prometheus",
         "hide": 0,
-        "includeAll": false,
-        "label": "Cluster",
-        "name": "cluster",
-        "query": "label_values(opensearch_cluster_status, cluster)",
-        "refresh": 1,
-        "regex": "",
-        "sort": 1,
-        "type": "query"
+        "includeAll": true,
+        "label": "Role",
+        "multi": true,
+        "name": "role",
+        "options": [
+          {
+            "selected": true,
+            "text": "All",
+            "value": "$__all"
+          },
+          {
+            "selected": false,
+            "text": "data",
+            "value": "data"
+          },
+          {
+            "selected": false,
+            "text": "cluster_manager",
+            "value": "cluster_manager"
+          },
+          {
+            "selected": false,
+            "text": "voting",
+            "value": "voting"
+          },
+          {
+            "selected": false,
+            "text": "coordinating_only",
+            "value": "coordinating_only"
+          },
+          {
+            "selected": false,
+            "text": "ingest",
+            "value": "ingest"
+          },
+          {
+            "selected": false,
+            "text": "ml",
+            "value": "ml"
+          },
+          {
+            "selected": false,
+            "text": "unrecognized",
+            "value": "unrecognized"
+          }
+        ],
+        "query": "data, cluster_manager, voting, coordinating_only, ingest, ml, unrecognized",
+        "refresh": 0,
+        "type": "custom"
       },
       {
         "datasource": {
           "type": "datasource",
           "uid": "${prometheusds}"
         },
         "hide": 0,
-        "includeAll": true,
-        "label": "Node",
```

Comment:
"Node" is not really useful, as we already can filter by juju-related tags like `juju_unit`